lib/osf-components/addon/components/file-browser/file-rename-modal/component.ts

```diff
@@ -32,7 +33,21 @@ export default class FileRenameModal extends Component<Args> {
 
     @action
     resetFileNameValue() {
-        this.newFileName =  this.originalFileName;
+        try {
+            return this.newFileName = this.originalFileName;
+
+        } catch (e) {
+            throw new NotUpdatedError('New file name not updated.');
+        }
+    }
+
+    updateOriginalName(newName: string) {
+        try {
+            return this.originalFileName = newName;
+
+        } catch (e) {
+            throw new NotUpdatedError('Original file name not updated.');
+        }
```

Comment:
I am not sure I understand the purpose of these error handling code here. Under what circumstances will updating `this.newFileName` or `this.originalFileName` result an error? Also what's the purpose of `throw`ing these errors here? Are there any upstream error handler catching these errors?

Reply:
Hey Yuhuai, these were initial logic statements but we can hold off if you think implementation is not yet warranted or necessary. We have had several bugs where the rename did not properly occur but may have fixed them without additional handling. The extension of the OsfError class from Error was more of the work that added value, and this was an opportunity to use it.